api.js

```diff
@@ -1332,34 +1331,16 @@ API.prototype.instrumentLoadedModule = function instrumentLoadedModule(moduleNam
     NAMES.SUPPORTABILITY.API + '/instrumentLoadedModule'
   )
   metric.incrementCallCount()
-  try {
-    const instrumentationName = shimmer.getInstrumentationNameFromModuleName(moduleName)
-    if (!shimmer.registeredInstrumentations[instrumentationName]) {
```

Comment:
I'm not sure this matters but `shimmer.instrumentPostLoad` will now only return false if instrumentation errored. this logic states it used to return false if there was no registered instrumentation or lacked an onRequire hook

Reply:
Good catch. You know, I think this would all be more correct if we just return 'false' unless it was actually instrumented... including onRequire() returning false.

While it wasn't doing that prior, I think that is more correct and would actually consider that a 'fix' over the previous behavior of returning 'true' even if onRequire() indicated it didn't instrument.

I'm going to update shimmer.instrumentPostLoad() to only return 'true' if flagged as instrumented.